charts/gardener/resource-manager/charts/runtime/templates/configmap.yaml

```
@@ -0,0 +1,149 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: gardener-resource-manager-config
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: gardener
+    role: resource-manager
+data:
+  config.yaml: |
+    ---
+    apiVersion: resourcemanager.config.gardener.cloud/v1alpha1
+    kind: ResourceManagerConfiguration
+    sourceClientConnection:
+      qps: {{ .Values.global.config.sourceClientConnection.qps }}
+      burst: {{ .Values.global.config.sourceClientConnection.burst }}
+    {{- if .Values.global.config.sourceClientConnection.namespace }}
+      namespace: {{ .Values.global.config.sourceClientConnection.namespace }}
+    {{- end }}
+    {{- if .Values.global.config.sourceClientConnection.cacheResyncPeriod }}
+      cacheResyncPeriod: {{ .Values.global.config.sourceClientConnection.cacheResyncPeriod }}
+    {{- end }}
+    {{- if .Values.global.config.sourceClientConnection.kubeconfig }}
+      kubeconfig: /etc/gardener-resource-manager/source-kubeconfig/kubeconfig
+    {{- end }}
+    {{- if .Values.global.config.targetClientConnection }}
+    targetClientConnection:
+      qps: {{ .Values.global.config.targetClientConnection.qps }}
+      burst: {{ .Values.global.config.targetClientConnection.burst }}
+    {{- if .Values.global.config.sourceClientConnection.namespace }}
+      namespace: {{ .Values.global.config.targetClientConnection.namespace }}
+    {{- end }}
+    {{- if .Values.global.config.targetClientConnection.disableCachedClient }}
+      disableCachedClient: {{ .Values.global.config.targetClientConnection.disableCachedClient }}
+    {{- end }}
+    {{- if .Values.global.config.targetClientConnection.cacheResyncPeriod }}
+      cacheResyncPeriod: {{ .Values.global.config.targetClientConnection.cacheResyncPeriod }}
+    {{- end }}
+    {{- if .Values.global.config.targetClientConnection.kubeconfig }}
+      kubeconfig: /etc/gardener-resource-manager/target-kubeconfig/kubeconfig
+    {{- end }}
+    {{- end }}
+    leaderElection:
+      leaderElect: {{ .Values.global.config.leaderElection.leaderElect }}
+      leaseDuration: {{ .Values.global.config.leaderElection.leaseDuration }}
+      renewDeadline: {{ .Values.global.config.leaderElection.renewDeadline }}
+      retryPeriod: {{ .Values.global.config.leaderElection.retryPeriod }}
+      resourceLock: {{ .Values.global.config.leaderElection.resourceLock }}
+      resourceName: {{ .Values.global.config.leaderElection.resourceName }}
+      resourceNamespace: {{ .Release.Namespace }}
+    logLevel: {{ .Values.global.config.leaderElection.logLevel }}
+    logFormat: {{ .Values.global.config.leaderElection.logFormat }}
```

`logLevel` and `logFormat` under `leaderElection` looks strange. Maybe something like this.
```suggestion
    logLevel: {{ .Values.global.config.logLevel | default "info" }}
    logFormat: {{ .Values.global.config.logFormat | default "json" }}
```